app_dart/lib/src/request_handlers/get_build_status.dart

```diff
@@ -27,13 +28,16 @@ class GetBuildStatus extends RequestHandler<Body> {
   final BuildStatusServiceProvider buildStatusProvider;
 
   static const String branchParam = 'branch';
+  static const String repoParam = 'repo';
 
   @override
   Future<Body> get() async {
     final DatastoreService datastore = datastoreProvider(config.db);
     final BuildStatusService buildStatusService = buildStatusProvider(datastore);
-    final String branch = request.uri.queryParameters[branchParam] ?? 'master';
-    final BuildStatus status = await buildStatusService.calculateCumulativeStatus(branch: branch);
+    final String branch = request.uri.queryParameters[branchParam] ?? config.defaultBranch;
+    final RepositorySlug repoSlug =
```

Comment:
Since we decided to move to repoSlug in the future, I'd like to go with that in this change and follow up with another change that will update the older occurrences.

Reply:
Okay, file a tech debt bug and assign to me